source/Symbol/SwiftASTContext.cpp

```diff
@@ -3947,12 +3956,12 @@ SwiftASTContext::GetTypeFromMangledTypename(const char *mangled_typename,
                 "not cached, searching",
                 this, mangled_typename);
 
-  std::string swift_error;
-  found_type = swift::ide::getTypeFromMangledSymbolname(
-                   *ast_ctx, mangled_typename, swift_error)
+  found_type = swift::Demangle::getTypeForMangling(*ast_ctx, mangled_typename)
                    .getPointer();
 
   if (found_type) {
+    found_type = convertSILFunctionTypesToASTFunctionTypes(found_type)
```

Comment:
Can you add a comment here?

Reply:
Good idea. I put the comment on the function itself